### ISSUE TITLE
Update logo on app launch screen

### ISSUE THREAD

**Issue by Popolechien** [kiwix/kiwix-android#2721]
For some reason the old logo is still showing when opening the app:
![Screenshot_20211020-142826](https://user-images.githubusercontent.com/21675885/138092753-66ff6427-12c4-4097-8942-26c0ec36c206.png)

The current version of our logo can be found here: https://commons.wikimedia.org/wiki/File:Logo-kiwix-vertical.svg (or [here](https://upload.wikimedia.org/wikipedia/commons/thumb/c/c4/Logo-kiwix-vertical.svg/512px-Logo-kiwix-vertical.svg.png) for the png)

**Comment by Popolechien** [comment]
https://commons.wikimedia.org/wiki/Category:Logos_of_Kiwix ?

**Comment by cdhiraj40** [comment]
we have the same old logo in left menu  too

**Comment by Popolechien** [comment]
@cdhiraj40 Yep - I'll open a separate ticket for that one once the new version is ready.

**Comment by cdhiraj40** [comment]
@kelson42 @gouri-panda I think we missed the starting screen after the app is installed or this is intentional?

![image](https://user-images.githubusercontent.com/75211982/143477458-6425f75f-ae26-4d89-b3b1-db901040f482.png)

**Comment by kelson42** [comment]
The logo is obviously still wrong indeed.

**Comment by cdhiraj40** [comment]
> The logo is obviously still wrong indeed.

should I create a ticket for it?

**Comment by kelson42** [comment]
@cdhiraj40 or you can just make directly a PR, sounds trivial.

**Comment by cdhiraj40** [comment]
> @cdhiraj40 or you can just make directly a PR, sounds trivial. But I wonder why we use different logo files in two different places of the app!

sure I will make a PR soon. :)
oh sorry about that question

**Comment by gouri-panda** [comment]
@kelson42 @cdhiraj40 We discussed a separate ticket for the menu's left screen logo, but we never thought about onboarding screen. @cdhiraj40 Thanks for pointing this out. Please go ahead!

**Comment by AdiAr11** [comment]
@gouri-panda is there something i can do as well?

**Comment by gouri-panda** [comment]
@AdiAr11 You can update the logo of the left menu.

**Comment by AdiAr11** [comment]
@gouri-panda sure. I'll start working on it. But I think some cropping was needed with the provided image.

**Comment by gouri-panda** [comment]
Use android studios crop funcanality. If it's not helpful, feel free to ask @Popolechien .

**Comment by AdiAr11** [comment]
okay, Thank you
